wkcuber/api/bounding_box.py

```diff
@@ -1,60 +1,83 @@
-from typing import Tuple, Union, List, Dict, Generator, Optional
+# mypy: allow-untyped-defs
 import json
+from typing import Dict, Generator, Iterable, List, Optional, Tuple, Union, NamedTuple
+
 import numpy as np
 from wkcuber.mag import Mag
 
 Shape3D = Union[List[int], Tuple[int, int, int], np.ndarray]
 
 
+class BoundingBoxNamedTuple(NamedTuple):
```

Comment:
In the code in the voxelytics repo this class belongs to `voxelytics.configs.datasources.schema.py` and was imported here. I added this class here to avoid that wkcuber gets dependent on voxelytics.

Reply:
Great! If you do the PR in voxelytics, you can remove BoundingBoxNamedTuple by importing it from wkcuber.